qemu/tests/thin_provisioning.py

```diff
@@ -0,0 +1,170 @@
+import re
+import os
+import logging
+
+from autotest.client.shared import error
+from autotest.client.shared import utils
+from autotest.client import os_dep
+from virttest import env_process
+from avocado.utils import process
+from avocado.core import exceptions
+
+
+def get_scsi_disk(session=None):
```

Comment:
If you think this script can cover test point of block_discard.py? can we drop block_discard.py or apply your modification into block_diskcard.py